rust/main/agents/relayer/src/msg/metadata/message_builder.rs

```diff
@@ -137,6 +137,7 @@ impl MessageMetadataBuilder {
 }
 
 /// Builds metadata for a message.
+#[instrument(err, skip(message_builder, message, params), fields(destination_domain=self.destination_domain().name()), ret)]
 pub async fn build_message_metadata(
     message_builder: MessageMetadataBuilder,
     ism_address: H256,
```

Comment:
you can skip `ism_address` as well and instead have it as a named `field` like `ism_address=ism_address.to_string()`, because H256 is printed as a byte array iirc. I'd remove the `ret` since the metadata result is very verbose and not human readable

Reply:
Done